sql/firefox_kpi_dashboard_v1.sql

```diff
@@ -39,24 +39,29 @@ function jackknife_resamples(arr) {
     }
     return output;
 }
+
 function array_sum(arr) {
     return arr.reduce(function (acc, x) {
         return acc + x;
     }, 0);
 }
+
 function array_avg(arr) {
     return array_sum(arr) / arr.length;
 }
-function sum_buckets_with_ci(counts_per_bucket) {
+
+function sum_buckets_with_ci(n_buckets, counts_per_bucket) {
+    if (counts_per_bucket.every(x => x == null)) {
+      return null;
+    };
```

Comment:
In smoot usage, we found that if the source fields we're aggregating over are null, we end up passing in an array of 20 null elements, which leads to NaNs.